$OpenBSD: patch-test_T355-smime_sh,v 1.2 2020/08/13 18:21:19 sthen Exp $

OpenSSL has sha1 as default fingerprint algo
LibreSSL has md5 as default fingerprint algo

Index: test/T355-smime.sh
--- test/T355-smime.sh.orig
+++ test/T355-smime.sh
@@ -6,7 +6,7 @@ test_description='S/MIME signature verification and de
 test_require_external_prereq openssl
 test_require_external_prereq gpgsm
 
-FINGERPRINT=$(openssl x509 -fingerprint -in "$NOTMUCH_SRCDIR/test/smime/key+cert.pem" -noout | sed -e 's/^.*=//' -e s/://g)
+FINGERPRINT=$(openssl x509 -fingerprint -sha1 -in "$NOTMUCH_SRCDIR/test/smime/key+cert.pem" -noout | sed -e 's/^.*=//' -e s/://g)
 
 add_gpgsm_home
 
